orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/OrchSessionService.java

```diff
@@ -77,6 +77,7 @@ public OrchSessionItem addOrUpdateSessionId(
                 newItem =
                         new OrchSessionItem()
                                 .withSessionId(newSessionId)
+                                .withAccountState(OrchSessionItem.AccountState.UNKNOWN)
```

Comment:
Does this also need to be added to the `addSession` method above this one?

Reply:
I wouldn't really expect `addSession` to modify the session object I pass in, but given it's only used in one place we could rename to `addNewSession` and do the refactoring, what do you think?

Reply:
Does `.withAccountState(OrchSessionItem.AccountState.UNKNOWN)` need adding [here](https://github.com/govuk-one-login/authentication-api/blob/d8f198ffffb1eb06e023ed35f879820ac940227b/oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandler.java#L515)?

Reply:
I think on refactoring we should probably do something similar to what @GHSwallow has done here https://github.com/govuk-one-login/authentication-api/pull/5384, where the new session constructor should have some defaults like `isNewAccount = UNKNOWN` similar to the current session class. Thoughts?

Reply:
yep that sounds like a good idea to me 👍

Reply:
Cool have added this